Source/IGListCollectionView.m

```diff
@@ -19,11 +19,25 @@ - (instancetype)initWithFrame:(CGRect)frame collectionViewLayout:(UICollectionVi
             self.backgroundColor = [UIColor whiteColor];
         }
         
-        self.alwaysBounceVertical = YES;
+        [self updateIGListCollectionViewDefaultValues];
     }
     return self;
 }
 
+- (instancetype)initWithCoder:(NSCoder *)aDecoder {
+    if (self = [super initWithCoder:aDecoder]) {
+        [self updateIGListCollectionViewDefaultValues];
+    }
+    return self;
+}
```

Comment:
ha 😅  we removed `initWithCoder:` previously so that we don't override storyboard settings.

let's remove, but let's keep your tests (asserts will need to be flipped)